lua/starfall/libs_sh/color.lua

```diff
@@ -155,6 +196,18 @@ function color_methods:hsvToRGB()
 	return wrap({ rgb.r, rgb.g, rgb.b, (rgb.a or 255) })
 end
 
+--- Returns a hexadecimal string representation of the color
+-- @param alpha Optional boolean whether to include the alpha channel, False by default
+-- @return String hexadecimal color
+function color_methods:toHex(alpha)
+	if alpha~=nil then checkluatype(alpha, TYPE_BOOL) end
+	if alpha then
+		return string.format("%X%X%X%X", self[1], self[2], self[3], self[4])
```

Comment:
`%X` should be replaced with `%02x` so there's always two hexadecimal digits per channel.

Reply:
Ah, dunno why I removed it

Reply:
How about `%02X`?

Reply:
Yeah I think uppercase is fine too.

Reply:
Nah, did google search and turns out many people prefer lowercase colors and many graphics software make them lowercase too, so I guess I was just one of the few that made them uppercase.
Ty for help.